### ISSUE TITLE
how change to my color theme?

### ISSUE THREAD

**Issue by longxianlin** [squimer/DatePickerDialog-iOS-Swift#12]
I want background black color, font color white
change many views, not working.

```swift
DatePickerDialog().show("选择预约日期", doneButtonTitle: "确定", cancelButtonTitle: "取消", datePickerMode: UIDatePickerMode.DateAndTime) {
                (date) -> Void in
                print("选中时间\(date)")
                self.pickedDate = date
                self.tableView.reloadData()
                //self.textField.text = "\(date)"
            }
```

this work in swift2.1

**Comment by longxianlin** [comment]
ok, 
i try this libs 
FxDatePicker, very good

**Comment by lfarah** [comment]
@HardcoreIOS said:
Please allow the user to have custom features. Like I want the background to have a blur when the picker view is presented, the pickerview's background to be a black instead of the default gray, and the pickerview's text's to be white instead of the default black!

**Comment by Reservations** [comment]
Is customizing the color possible yet?

**Comment by lfarah** [comment]
Not yet. I didn't have time to work on this project lately, but I didn't forget about you! @Reservations

**Comment by benjohnde** [comment]
That sounds great. Could be a bit difficult to let the DatePickerDialog be fully customizable. But it should be worth to implement a few styles (for instance dark-blurry, white-transparent and so on and so forth).

**Comment by aravasio** [comment]
No news on this yet, right?

**Comment by Reservations** [comment]
@aravasio I ended up just customizing the color by editing the RGB settings in the code of the library.

**Comment by aravasio** [comment]
Good call. For some reason it never occurred to me to actually modify a Pod.

**Comment by lfarah** [comment]
Hey @Reservations, what do you think about opening a PR so other people can customize the colors?

**Comment by lfarah** [comment]
Really sorry guys for not giving attention to this... I'm really packed with stuff to do in other projects

**Comment by Vidhanshu** [comment]
Just change the gradient color or remove the gradient and change background colors of all title label, Done button, date picker and cancel button.
